pyluna-pathology/luna/pathology/analysis/qupath/import_regional_geojson_into_qupath.groovy

```diff
@@ -9,11 +9,10 @@ import java.nio.file.Paths;
 import java.util.stream.Collectors;
 import qupath.lib.geom.Point2
 
-
-
-def imageData = getCurrentImageData()
-def filename = GeneralTools.getNameWithoutExtension(imageData.getServer().getMetadata().getName())
-def image_id = filename.replace(".svs", "").toString()    
+def imageData = QPEx.getCurrentImageData()
+def server = getCurrentServer()
+def filename = GeneralTools.getNameWithoutExtension(server.getMetadata().getName())
+def image_id = filename.replace(".svs", "").toString()  
```

Comment:
Better to substring and remove the last extension bit, if QuPath supports other file formats